gobblin-test-harness/src/test/java/org/apache/gobblin/TaskErrorIntegrationTest.java

```diff
@@ -232,12 +231,16 @@ public void shutdown(SourceState state) {
     }
   }
 
-  private class TestAppender extends AppenderSkeleton {
-    List<LoggingEvent> events = new ArrayList<LoggingEvent>();
-    public void close() {}
-    public boolean requiresLayout() {return false;}
+  public class TestAppender extends AbstractAppender {
```

Comment:
Can we use one class of TestAppender instead of creating one in different test classes?
Definitely need not be the part of this PR.

Reply:
ya, since it was getting big, wanted to avoid more refactor, I will create minor ticket for this.